packages/preferences/src/browser/style/index.css

```diff
@@ -23,3 +23,126 @@
     display: flex;
     line-height: var(--theia-content-line-height) !important;
 }
+
+/* UI View */
+
+@import url("./context-menu.css");
+@import url("./navbar.css");
+@import url("./preference-array.css");
+@import url("./preference-object.css");
+@import url("./search-input.css");
+
+:root {
+  --theia-settingsSticky-height: 100px;
+}
+
+html,
```

Comment:
please don't touch global styles

Reply:
I changed the variables to only be under the settings container.

Reply:
please get rid of styles to global element like html and body